### ISSUE TITLE
[Frontend] - Remove page blocks on the Admin Tools User Management tab

### ISSUE THREAD

**Issue by AdityaBoddepalli** [Northeastern-Electric-Racing/FinishLine#2140]
### Description

(Blocked by #2128)
**Important:** Branch off and merge into the feature/admin-tools-redesign branch

The admin tools on the user management tab are wrapped in page blocks. Remove the page blocks for all the tools on that tab.
It should look similar to the mocks below when you are done. Keep in mind the mocks aren't all encompassing, they are to give you a feel as to what the page should look like.

### Acceptance Criteria

- [ ] The functionality of the admin tools should remain the same, especially the visibility of tools based on user role.
- [ ] There shouldn't be any page blocks on the user management tab.
- [ ] The user management tab should include the Role Management and Team management tools

### Proposed Solution

N/A

### Mocks

![image](https://github.com/Northeastern-Electric-Racing/FinishLine/assets/68555142/ccd50211-8fcd-4d8e-b423-d58b92b074f5)
